### PR TITLE
doc: add 0.7 release notes

### PR DESCRIPTION
**Pull request by dbkinder**
Review draft of 0.7 release notes

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

File: doc/release_notes_0.7.rst [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+************************
+
+Enable cache QOS with CAT
+=========================
+
+Cache Allocation Technology (CAT) is enabled on Apollo Lake (APL)
+platforms, providing cache isolation between VMs mainly for real-time
+performance quality of service (QoS).  The CAT for a specific VM is
+normally set up at boot time per the VM configuration determined at
+build time. For debugging and performancing tuning, the CAT can also be
```
Comment (gvancuts) [inline doc/release_notes_0.7.rst]:
```suggestion
build time. For debugging and performance tuning, the CAT can also be
```

File: doc/release_notes_0.7.rst [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+enabled and configured at runtime by writing proper values to certain
+MSRs using the ``wrmsr`` command on ACRN shell.
+
+Support ACPI power key mediator
+===============================
+ACRN supports ACPI power/sleep key on the APL and KBL NUC platforms,
```
Comment (gvancuts) [inline doc/release_notes_0.7.rst]: Is this specific to APL and KBL or is it more a case of SBL vs. EFI?

File: doc/release_notes_0.7.rst [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+- :acrn-issue:`2548` - [APLNUC/KBLNUC][GVT][SOS/LAAG] Weston fails to play video in SOS and UOS
+- :acrn-issue:`2572` - Startup SOS Fails
+- :acrn-issue:`2588` - Uninitialized Variable is used in acrn_kernel/drivers/acrn/acrn_trace.c and acrn_hvlog.c
+- :acrn-issue:`2606` - HV crash during running VMM related Hypercall fuzzing test.
+- :acrn-issue:`2624` - Loading PCI devices with table_count > CONFIG_MAX_MSIX_TABLE_NUM leads to writing outside of struct.
+- :acrn-issue:`2643` - Ethernet pass-through, network card can't get ip in uos
+- :acrn-issue:`2674` - VGPU needs the lock when updating ppggt/ggtt to avoid the race condition
+- :acrn-issue:`2695` - UOS powers off or suspend while pressing power key, UOS has no response
+
+Known Issues
```
Comment (gvancuts) [inline doc/release_notes_0.7.rst]: What is the difference betwen "New Issues" and "Known Issues"

File: doc/release_notes_0.7.rst [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+
+   **Impact:** Power Management state-related operations in SOS/UOS on
+   KBL NUC can't be used.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`2279` - [APLNUC] After exiting UOS with mediator
+   Usb_KeyBoard and Mouse, SOS cannot use the USB key board and mouse.
```
Comment (gvancuts) [inline doc/release_notes_0.7.rst]:
```suggestion
   Usb_KeyBoard and Mouse, SOS cannot use the USB keyboard and mouse.
```

File: doc/release_notes_0.7.rst [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+
+   Thise steps reproduce the issue:
```
Comment (gvancuts) [inline doc/release_notes_0.7.rst]:
```suggestion
   These steps reproduce the issue:
```

File: doc/release_notes_0.7.rst [hunk 6]
```diff
@@ -0,0 +1,382 @@
...
+
+   1) Insert USB keyboard and mouse in standard A port (USB3.0 port)
+   2) Boot UOS by sharing the USB keyboard and mouse in cmd line:
+
+      ``-s n,xhci,1-1:1-2:1-3:1-4:2-1:2-2:2-3:2-4 \``
+
+   3) UOS access USB keyboard and mouse.
+   4) Exit UOS.
+   5) SOS access USB keyboard and mouse.
```
Comment (gvancuts) [inline doc/release_notes_0.7.rst]: Shouldn't that be "SOS cannot access..."

File: doc/release_notes_0.7.rst (hunk 3 above)
Comment (dbkinder) [reply doc/release_notes_0.7.rst]: Not sure.  I'll just merge these two lists I got from Nanlin

File: doc/release_notes_0.7.rst (hunk 2 above)
Comment (dbkinder) [reply doc/release_notes_0.7.rst]: @NanlinXie ?

File: doc/release_notes_0.7.rst (hunk 3 above)
Comment (NanlinXie) [reply doc/release_notes_0.7.rst]: @dbkinder @gvancuts New feature list is the issues with type "feature/story", and know issues are with the type "bug". Suggest to split these two list as previous release did.

Comment (dbkinder) [reply doc/release_notes_0.7.rst]: OK, I separated the lists as they were.

File: doc/release_notes_0.7.rst (hunk 2 above)
Comment (NanlinXie) [reply doc/release_notes_0.7.rst]: @gvancuts It is for  UEFI platform, OpenSource feature.

Comment (gvancuts) [reply doc/release_notes_0.7.rst]: Thanks @NanlinXie , is therefore true that UP2 with SBL is not support but that UP2 running EFI will work (just like the NUCs)?

If so, I would suggest to reword this to indicate that we support these on EFI platforms.
